packages/platform/libraries/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@calcom/platform-libraries",
-  "version": "0.0.0",
+  "version": "1.2.3",
```

Comment:
This is temporary, will fix it once we release a new version of platform libraries.